### PR TITLE
Feature/sentry updates

### PR DESCRIPTION
**Pull request by ben-vr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "lint"
   ],
   "dependencies": {
+    "@sentry/node": "^4.5.3",
```
Comment (knor-el-snor) [inline package.json]: Use ~

File: src/server.ts [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-import { logger, raven } from './lib/logger';
+import { logger, sentry } from './lib/logger';
 import { importTranslations } from './lib/translator';
 import { app } from './app';
 import { errorTranslations } from './constants';
 
-raven.config(process.env.SENTRY_DSN, {
-  environment: process.env.NODE_ENV,
-}).install();
+sentry.init({ dsn: process.env.SENTRY_DSN });
```
Comment (knor-el-snor) [inline src/server.ts]: How does Sentry make the difference between environments now? With the same `SENTRY_DSN` as before?

File: tests/test.config.ts [hunk 3]
```diff
@@ -21,10 +21,6 @@ Object.keys(environment).forEach((key) => {
   process.env[key] = environment[key];
 });
 
-// Disable Sentry warnings
-import * as raven from 'raven';
```
Comment (knor-el-snor) [inline tests/test.config.ts]: Is there an alternative for handling the logs during tests with the new package?

File: src/server.ts (hunk 2 above)
Comment (ben-vr) [reply src/server.ts]: Sentry.init({
  environment: 'staging',
})

https://docs.sentry.io/enriching-error-data/environments/?platform=node

Adding environment config to sentry

**Pull request merged**